docs/configuration.asciidoc

```diff
@@ -190,7 +190,7 @@ Note that not all core Node.js API's can be instrumented without the use of Asyn
 
 * *Type:* Array
 * *Default:* `[]`
-* *Env:* `ELASTIC_TRANSACTION_IGNORE_URLS`
+* *Env:* `ELASTIC_APM_TRANSACTION_IGNORE_URLS`
 
```

Comment:
Yes. I had to go through the Python (hasn't documented transaction_ignore_urls), Ruby (also hasn't doc'd it), to the Java agent which *has* doc'd it to compare.

I'm happy to move this to a separate PR if you prefer. I feel a little bad merging in somewhat unrelated changes to the same commit.

> might be a low hanging spec PR win for the future.

True, it would be good to spec the envvars.

Reply:
Sergey pointed out https://docs.google.com/spreadsheets/u/1/d/1JJjZotapacA3FkHc2sv_0wiChILi3uKnkwLTjtBmxwU/preview#gid=0 to me earlier today, FWIW.